terragrunt/env/central/central_account/terragrunt.hcl

```diff
@@ -0,0 +1,15 @@
+terraform {
+  source = "../../../aws//central_account"
+}
+
+inputs = {
+  bucket_name = "cbs-central-satellite-${get_aws_account_id()}"
+  cbs_central_bucket_arn = "arn:aws:s3:::cbs-central-log-archive-339850311124"
```

Comment:
What would you think of using `get_aws_account_id()` here since I think it'll always be this account ID running this module:
```hcl
cbs_central_bucket_arn = "arn:aws:s3:::cbs-central-log-archive-${get_aws_account_id()}"
```

Reply:
thats a great idea! Easier to pivot in the future if our account numbers ever change for whatever reason